app/assets/stylesheets/application.scss

```diff
@@ -15,3 +15,30 @@
  *= require foundation_and_overrides
 
  */
+  @font-face {
+    font-family: "MKXfont";
+    src: font-url('MKXfont.ttf');
+  }
+
+  body {
+    background-image: no-repeat;
+    background-image: url('MKX-wallpaper.png');
+    background-attachment: fixed;
+    background-position: 50% 28.5%;
+    position: relative;
+  }
+
+  .name {
+    display: inline-block;
+    height: 25px;
+  }
+
+  .top-bar .name h4 a:before {
```

Comment:
Line should be indented 0 spaces, but was indented 2 spaces<br>Selector should have depth of applicability no greater than 2, but was 4